### PR TITLE
[feat] Update cli.ts, add `-c` support absolute path

### PR DESCRIPTION
**Pull request by zhangfisher**
The - c parameter of tsoa does not support specifying an absolute path

```shell
tsoa spec-and-routes -c  c:/work/code/mylib/tsoa.json // not work
```

this commit is add support  specifying an absolute path.

```shell
tsoa spec-and-routes -c  c:/work/code/mylib/tsoa.json // now it is work
```

 

### All Submissions:

- [x] Have you followed the guidelines in our [Contributing](https://github.com/lukeautry/tsoa/tree/master/docs/CONTRIBUTING.md) document?
- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/lukeautry/tsoa/pulls) for the same update/change?
- [ ] Have you written unit tests?
- [ ] Have you written unit tests that cover the negative cases (i.e.: if bad data is submitted, does the library respond properly)?
- [ ] This PR is associated with an existing issue?

**Closing issues**

Put `closes #XXXX` (where XXXX is the issue number) in your comment to auto-close the issue that your PR fixes.

### If this is a new feature submission:

- [ ] Has the issue had a maintainer respond to the issue and clarify that the feature is something that aligns with the [goals](https://github.com/lukeautry/tsoa#goal) and [philosophy](https://github.com/lukeautry/tsoa#philosophy) of the project?

**Potential Problems With The Approach**

**Test plan**

### CODE REVIEW

File: tests/pnpm-lock.yaml [hunk 1]
```diff
@@ -0,0 +1,8014 @@
+lockfileVersion: '9.0'
```
Comment (WoH) [inline tests/pnpm-lock.yaml:1]: Remove file

**Comment by WoH** [comment]
Are you sure these tests did not pass before making the code change?
Haven't checked yet, but it seems like they may be unrelated

**Comment by zhangfisher** [comment]
I have new unrelated submissions before the PR is accepted

It is automatic and can be ignored

By the way, PR review is too slow

**Comment by WoH** [comment]
I assumed these trivial mistakes (unrelated commits, superfluous artifacts, incorrect types/failing CI) would be noticed during the self-review the PR

File: pnpm-lock.yaml [hunk 2]
```diff
@@ -0,0 +1,5873 @@
+lockfileVersion: '9.0'
```
Comment (WoH) [inline pnpm-lock.yaml:1]: Please remove this file from source control

**Pull request closed without merging**